sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md

```diff
@@ -0,0 +1,119 @@
+Similar to how SYCL_DEVICE_FILTER applies filtering to the entire process this device selector can be used to
+programmatically select the Level-Zero backend.
+                
+When neither the environment variable nor the filtering device selector are used, the implementation chooses
+the Level-Zero backend for GPU devices supported by the installed Level-Zero runtime.
+The serving backend for a SYCL platform can be queried with the ```get_backend()``` member function of ```cl::sycl::platform```.
+
+## 4. Interoperability with the Level-Zero API
+
+This chapter describes the various interoperability possible between SYCL and Level-Zero.
+The application must include both of the following headers in order to use any of the interoperation APIs described in this section,
+and they must be included in the order shown:
+
+``` C++
+  #include "level_zero/ze_api.h"
+  #include "sycl/backend/level_zero.hpp"
+```
+### 4.1 Mapping of SYCL objects to Level-Zero handles
+
+These SYCL objects encapsulate the corresponding Level-Zero handles:
+|  SYCL object  |    Level-Zero handle   |
+|-------------|:------------|
+|platform |ze_driver_handle_t|
+|device   |ze_device_handle_t|
+|context  |ze_context_handle_t|
+|queue    |ze_command_queue_handle_t|
+|program  |ze_module_handle_t|
+
+### 4.2 Obtaining of native Level-Zero handles from SYCL objects
+                
+The ```get_native<cl::sycl::backend::level_zero>()``` member function is how a raw native Level-Zero handle can be obtained
+for a specific SYCL object. It is currently supported for SYCL ```platform, device, context, queue, event``` and ```program``` classes.
+There is also a free-function defined in ```cl::sycl``` namespace that can be used instead of the memeber function:
+``` C++
+template <backend BackendName, class SyclObjectT>
+auto get_native(const SyclObjectT &Obj) ->
+	typename interop<BackendName, SyclObjectT>::type;
+```
+### 4.3 Construct a SYCL object from a Level-Zero handle
+        
+The following free functions defined in the ```cl::sycl::level_zero``` namespace allow an application to create
+a SYCL object that encapsulates a corresponding Level-Zero object:
+
+| Level-Zero interoperability function |Description|
+|-------------|:------------|
+|``` make<platform>(ze_driver_handle_t);```|Constructs a SYCL platform instance from a Level-Zero ```ze_driver_handle_t```.|
+|``` make<device>(const platform &, ze_device_handle_t);```|Constructs a SYCL device instance from a Level-Zero ```ze_device_handle_t```. The platform argument gives a SYCL platform, encapsulating a Level-Zero driver supporting the passed Level-Zero device.|
+|``` make<context>(const vector_class<device> &, ze_context_handle_t);```| Constructs a SYCL context instance from a Level-Zero ```ze_context_handle_t```. The context is created against the devices passed in. There should be at least one device given and all the devices must be from the same SYCL platform and thus from the same Level-Zero driver.|

```

Comment:
```suggestion
|``` make<context>(const vector_class<device> &, ze_context_handle_t);```| Constructs a SYCL context instance from a Level-Zero ```ze_context_handle_t```. The context is created against the devices passed in. There must be at least one device given and all the devices must be from the same SYCL platform and thus from the same Level-Zero driver.|
```

I assume this is a requirement, right?  If so, use "must".

Reply:
OK.